chainer/training/extensions/step_shift.py

```diff
@@ -0,0 +1,87 @@
+from __future__ import division
+
+import numpy
+
+from chainer.training import extension
+
+
+class StepShift(extension.Extension):
+
+    """Trainer extension to shift an optimizer attribute in "steps".
+
+    This extension multiplies the specified attribute of the optimizer in
+    "steps". The typical use case is to scale the attribute at every ``k``th
```

Comment:
Thank you for the fix.
It seems ``` ``k``th ``` should be replaced with ``` ``k``\ th ``` here.

Reply:
By the way, this is an error in documentation.
Please check out [here](https://docs.chainer.org/en/stable/contribution.html#documentation) for details.